.github/workflows/integration_test.yaml

```diff
@@ -108,6 +108,12 @@ jobs:
         with:
           pattern: ${{ inputs.artifact-prefix }}-*
           merge-multiple: true
+      # TODO Remove pinned image when problem is resolved on jammy
+      - name: Pin arm lxc image for jammy
+        if: ${{ contains(matrix.job.spread_job, 'arm') }}
+        run: |
+          sudo lxc image copy ubuntu:3e7ef486d7a6 local:
+          sudo lxc image alias create "juju/ubuntu@22.04/arm64" 3e7ef486d7a6
```

Comment:
Pinning instead of switching to the jammy runner, because switching the runner will require making the tests compatible with jammy's python version (pyproj limits, linters etc.).